package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "invoiceplane",
-  "version": "1.5.9",
+  "version": "1.5.10-next",
```

Comment:
Shouldn't it be just `1.5.10`?

Reply:
Yes it should. Fixed by https://github.com/InvoicePlane/InvoicePlane/commit/2bdc3ad6359729d8cb76b0e5f63e9d9836259c67